policyengine_us/variables/gov/states/mt/tax/income/credits/mt_capital_gain_credit.py

```diff
@@ -0,0 +1,18 @@
+from policyengine_us.model_api import *
+
+
+class mt_capital_gain_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Montana capital gain credit"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://rules.mt.gov/gateway/RuleNo.asp?RN=42%2E4%2E502"
+    defined_for = StateCode.MT
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.mt.tax.income.credits.capital_gain
+
+        net_capital_gain = tax_unit("net_capital_gain", period)
```

Comment:
Should this be a negative credit if they have capital losses? Please add a test for this and cap at zero if needed. Or if the capping occurs in net_capital_gain (I'm on my phone so can't check) please add a comment noting that.

Reply:
I will leave a comment # The net capital gain variable is capped at 0